lib/ArduinoNative/src/WiFiUdp.h

```diff
@@ -45,11 +45,14 @@
 
 #include <Arduino.h>
 #include "IPAddress.h"
+#include <netinet/in.h>
 
 /******************************************************************************
  * Macros
  *****************************************************************************/
 
+#define MAXLINE 1024
```

Comment:
My comment: There is no comment here. 
C++ code should not use #define but static const data.

Reply:
Changed to private constant.